### PR TITLE
{lyn7352} adding more logging to mock asset builder test

### PR DESCRIPTION
**Pull request by jackalbe**
Signed-off-by: Allen Jackson <23512001+jackalbe@users.noreply.github.com>

### CODE REVIEW

**Review by amzn-mike — APPROVED** [review]

File: AutomatedTesting/Gem/PythonTests/PythonAssetBuilder/AssetBuilder_test_case.py [hunk 1]
```diff
@@ -10,21 +10,25 @@
 import azlmbr.math
 import azlmbr.legacy.general
 
-def raise_and_stop(msg):
-    print (msg)
-    azlmbr.editor.EditorToolsApplicationRequestBus(azlmbr.bus.Broadcast, 'ExitNoPrompt')
+print('Starting mock asset tests')
+azlmbr.legacy.general.idle_enable(True)
+azlmbr.legacy.general.idle_wait(1.0)
```
Comment (cgalvan) [inline AutomatedTesting/Gem/PythonTests/PythonAssetBuilder/AssetBuilder_test_case.py:15]: What is the wait for? Is there some kind of notification we can listen for instead?

Comment (jackalbe) [reply AutomatedTesting/Gem/PythonTests/PythonAssetBuilder/AssetBuilder_test_case.py:15]: the logs for the Editor.log is stopping right before starting up the RHI so I want the Editor to get cooking then do a the check to make sure all the main systems are ready

Comment (cgalvan) [reply AutomatedTesting/Gem/PythonTests/PythonAssetBuilder/AssetBuilder_test_case.py:15]: Spoke offline, I think we can listen for the ```NotifyMainWindowInitialized``` from ```AzToolsFramework::EditorEvents```. If that doesn't work, then we can add a new event that can be broadcast at the very end of the Editor startup.

Comment (jackalbe) [reply AutomatedTesting/Gem/PythonTests/PythonAssetBuilder/AssetBuilder_test_case.py]: Updated

**Review by cgalvan — APPROVED** [review]

**Pull request merged**